src/app/pages/Collectibles/components/AnimatedSvg.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { CSSProperties, FC, useState } from 'react';
+
+import { emptyFn } from 'app/utils/function.utils';
+
+interface Props {
+  uri: string;
+  alt?: string;
+  loader?: React.ReactElement;
+  className?: string;
+  style?: CSSProperties;
+  onError?: EmptyFn;
+}
+export const AnimatedSvg: FC<Props> = ({ uri, alt, loader, className, style, onError = emptyFn }) => {
+  const [isLoading, setIsLoading] = useState(true);
+
+  return (
+    <>
+      <img
+        src={uri}
+        alt={alt}
+        className={className}
+        style={style}
+        onLoad={() => setIsLoading(false)}
```

Comment:
Please wrap all new functions with no dependencies but for state setting callbacks into `useCallback`